scripts/delayed.js

```diff
@@ -1 +1,51 @@
 // add delayed functionality here
+function initBackToTop() {
+  const scrollThreshold = 1600;
+
+  const footerWrapper = document.querySelector('footer.footer-wrapper');
+  const footerBlock = footerWrapper.querySelector('.footer.block');
+
+  const createElement = (tag, classNames, textContent = '') => {
```

Comment:
`/scripts/dom-helpers.js` already exports several functions for creating desired elements. Please review `dom-helpers.js` and re-use one of the available methods